pkg/services/apiserver/config.go

```diff
@@ -43,6 +43,7 @@ func applyGrafanaConfig(cfg *setting.Cfg, features featuremgmt.FeatureToggles, o
 
 	o.KubeAggregatorOptions.ProxyClientCertFile = apiserverCfg.Key("proxy_client_cert_file").MustString("")
 	o.KubeAggregatorOptions.ProxyClientKeyFile = apiserverCfg.Key("proxy_client_key_file").MustString("")
+	o.KubeAggregatorOptions.RemoteServicesFile = apiserverCfg.Key("legacy_client_cert_auth").MustString("")
```

Comment:
is `o.KubeAggregatorOptions.RemoteServicesFile` correct?

Reply:
oh its not, copy paste error. Thanks for catching.